cfme/configure/configuration/region_settings.py

```diff
@@ -1082,7 +1082,11 @@ def get_replication_status(self, replication_type='global', host=None):
         if replication_type == 'remote':
             return view.is_displayed
         else:
-            return self._global_replication_row(host).is_displayed
+            is_host_present = self._global_replication_row(host).is_displayed
```

Comment:
This function name is really misleading, but you aren't changing the return type/behavior here. Just voicing my concern/opinion here - if you would like to improve it, that's great, we can discuss some better properties/methods for fetching replication status, or whether a host is configured for replication.